src/panels/config/voice-assistants/debug/assist-render-pipeline-run.ts

```diff
@@ -12,30 +12,42 @@ import { formatNumber } from "../../../../common/number/format_number";
 import "../../../../components/ha-yaml-editor";
 import { showAlertDialog } from "../../../../dialogs/generic/show-dialog-box";
 
-const RUN_DATA = {
-  pipeline: "Pipeline",
-  language: "Language",
-};
-const WAKE_WORD_DATA = {
-  engine: "Engine",
-};
-
-const STT_DATA = {
-  engine: "Engine",
-};
-
-const INTENT_DATA = {
-  engine: "Engine",
-  language: "Language",
-  intent_input: "Input",
-};
-
-const TTS_DATA = {
-  engine: "Engine",
-  language: "Language",
-  voice: "Voice",
-  tts_input: "Input",
-};
+const RUN_DATA = (hass: HomeAssistant) => ({
+  pipeline: hass.localize(
+    "ui.panel.config.voice_assistants.debug.stages.pipeline"
+  ),
+  language: hass.localize(
+    "ui.panel.config.voice_assistants.debug.stages.language"
+  ),
+});
```

Comment:
```suggestion
const RUN_DATA = ["pipeline", "language];
```

And then in `renderData` we can do : 

```ts
const label = hass.localize(
  `ui.panel.config.voice_assistants.debug.stages.${key}`
);
```